src/main/java/org/junit/internal/Throwables.java

```diff
@@ -39,4 +42,14 @@ public static Exception rethrowAsException(Throwable e) throws Exception {
     private static <T extends Throwable> void rethrow(Throwable e) throws T {
         throw (T) e;
     }
+
+    /**
+     * @return a String with the stacktrace of the given throwable
```

Comment:
Style-nit: remove the `@return` and instead give the method a description. The first sentence of a method's description shows up in the method summary section of the class Javadoc.